UnityProject/Assets/Scripts/Tilemaps/Utils/Types.cs

```diff
@@ -59,7 +59,8 @@ public static class LTSUtil
 	public static bool IsLayerIn(LayerTypeSelection SpecifyLayers, LayerType Layer)
 	{
 		LayerTypeSelection LayerCon = LayerType2LayerTypeSelection(Layer);
-		return (SpecifyLayers.HasFlag(LayerCon));
+		//HasFlag causes boxing, since we are only checking for one layer, a simple bit check is fine here
```

Comment:
instead of explaining why it isn't using HasFlag, this comment should explain what the line is doing for contributors who don't know bit operations that well.